canonical-web-and-design/ubuntu.com
# Update Kubernetes docs for 1.16

## Done

- added new pages to site nav
- update side navigation
- Update CDK -> Charmed Kubernetes
- Various doc updates for 1.16

## QA

- Check out this feature branch
- Run the site using the command ./run serve
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following QA steps
